app/plugin.go

```diff
@@ -502,6 +502,10 @@ func (a *App) GetMarketplacePlugins(filter *model.MarketplacePluginFilter) ([]*m
 
 		result = append(result, &model.MarketplacePlugin{
 			BaseMarketplacePlugin: &model.BaseMarketplacePlugin{
+				Labels: []model.MarketplaceLabel{{
+					Name:        "Local",
+					Description: "This plugin is not listed in the marketplace but was installed manually.",
+				}},
```

Comment:
This should be added only if the `EnableRemoteMarketplace` is enabled though. However, this setting is being introduced in this PR: https://github.com/mattermost/mattermost-server/pull/13449

What should be the best approach here to synch this? After you merge this maybe I can update the other PR to add the condition there

Reply:
Well, let's see what PR goes in first :wink: When can update the other one them.

Reply:
@marianunez I've updated the https://github.com/mattermost/mattermost-server/pull/13449 in https://github.com/mattermost/mattermost-server/pull/13449/commits/1035471dd138bc9865f367a4d936236e8f86fefa. I hope this was fine to you.